README.md

```diff
@@ -377,7 +377,12 @@ bower.
 ## Deployment: ##
 ----------
 This is fairly complex and is under active change as we have more projects deploy their own servers with various configurations.
-So I have moved it to its own wiki page:
-https://github.com/e-mission/e-mission-server/wiki/Deploying-your-own-server-to-production
+So I have moved it to the e-mission-server section in the e-mission-docs repo:
+https://github.com/e-mission/e-mission-docs/blob/master/docs/e-mission-server/deploying_your_own_server_to_production.md
 
 [Python_Structure]: https://raw.github.com/amplab/e-mission-server/master/figs/e-mission-server-module-structure.png
+
+## Wiki: ##
+----------
+All wikis have been moved to their own repository [e-mission-docs](https://github.com/e-mission/e-mission-docs). E-mission server specific wikis can be found here:
```

Comment:
Should this be at the end or at the beginning?
Also, maybe instead of "all wikis", say "additional documentation", since the new repository is not really a wiki any more :)

Reply:
Sure thing! Will put an "Additional Documentation" section in the beginning of the readme :)